fetchai/ledger/transaction.py

```diff
@@ -119,11 +135,7 @@ def signers(self):
         return self._signers
 
     def add_transfer(self, address: Identifier, amount: int):
-        assert amount > 0
```

Comment:
Are 0 value transfers valid? I don't think the original assert statement is the way to handle them if not, but we could silently ignore them?

Reply:
As far as I know, there is no limitation on Ledger side which would be enforcing `amount > 0`.

Reply:
As we just discussed, it's worth considering the possibility that many 0 value transfers could be used in a form of attack, and how to de-incentivize this.

Reply:
Good spot. Thanks for bringing this up!
I just checked the ledger code and thankfully this attack vector is already covered by fee economics involved in transfers handling - as by now there is constant fee charge per *each* transfer carried by transaction.